src/components/app/details/appConfig/AppConfig.tsx

```diff
@@ -472,9 +480,16 @@ function Navigation({
 }: AppConfigNavigationProps) {
     const location = useLocation()
     const selectedNav = navItems.filter((navItem) => location.pathname.indexOf(navItem.href) >= 0)[0]
+    const isGitOpsConfigurationRequired = navItems.find((item) => item.stage === STAGE_NAME.GITOPS_CONFIG)?.required
```

Comment:
this should be coming from parent as we are calculating this there

Reply:
Resolved